src/components/MyKiva/MyKivaProgressCard.vue

```diff
@@ -0,0 +1,169 @@
+<template>
+	<div class="progress-card">
+		<div class="tw-relative tw-z-docked">
+			<KvProgressCircle
+				class="tw-z-2"
+				:stroke-width="20"
+				:value="goalProgressPercentage"
+				:max="goalLoans"
+				:rotate="180"
+				style="height: 72px; width: 72px;"
+				:color="cardColor"
+			/>
+			<div
+				class="tw-absolute tw-flex tw-flex-col tw-items-center tw-justify-center tw-inset-0"
+				:class="{'tw-mt-0.5' : goalCompleted && !isAnnualGoal}"
+				style="height: 72px; width: 72px;"
```

Comment:
Could we have a const / computed for this and the one above? As same styling.

Reply:
You can take advantage of this: https://vuejs.org/guide/essentials/class-and-style#binding-inline-styles

Reply:
No worries, this is the exact size to fit within the ring